README.md

```diff
@@ -68,8 +35,4 @@ Give a ⭐️ if you like this project!
 - Inspiration
 - etc
 
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please add this License section in your readme file because all your projects should have a license file :+1:
- [ ] Also, add a license file in your repository you can add using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:
- [ ] Don't forget to add your name, current year, and app type in your MIT.md file after adding it in your project :+1: